src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1892,6 +1893,48 @@ export function registerTerminalActions() {
 			accessor.get(ITerminalService).doWithActiveInstance(t => t.clearBuffer());
 		}
 	});
+	registerAction2(class extends Action2 {
+		constructor() {
+			super({
+				id: TerminalCommandId.ShowWordLinkQuickpick,
+				title: { value: localize('workbench.action.terminal.showWordLinkQuickpick', "Show Word Link Quick Pick"), original: 'Show Word Link Quickpick' },
+				f1: true,
+				category,
+				precondition: ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated),
+			});
+		}
+		run(accessor: ServicesAccessor) {
+			accessor.get(ITerminalService).doWithActiveInstance(t => t.showLinkQuickpick(TerminalLinkProviderType.Word));
+		}
+	});
+	registerAction2(class extends Action2 {
+		constructor() {
```

Comment:
better names?

Reply:
How about: `Terminal: Open Detected Link`, `Terminal: Open Detected Link (Web)`, `Terminal: Open Detected Link (Local)`

I think we should do away with the word link provider in favor of the general `Terminal: Open Detected Link` which gathers links from all providers.